src/common/modal/Modal.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import ReactModal from 'react-modal';
+import classNames from 'classnames';
+
```

Comment:
Should we "grey out" the background when the modal is open? That is quite widely used effect with modals.

Reply:
I added a basic darkened background, but let's verify the design